docs/build/wasm/swanky.md

```diff
@@ -287,15 +293,20 @@ Swanky Node is a Substrate based blockchain configured to enable `pallet-contrac
 Swanky Node is optimized for local development, while removing unnecessary components such as P2P. Additional features and pallets, such as to interact between (Contract <-> Runtime), will be added in the future.
 
 ### Compatible ink! version
+
 ink! `v4.0.0-beta` or lower is supported by pallet-contract polkadot-0.9.33 branch.
```

Comment:
isn't 4.0.0 supported

Reply:
The 1.0.0 version is built with polkadot-0.9.33, and is referencing ink! v4.0.0-beta.
@shunsukew Does it make sense to make a new release?

But for the purpose of this document, it is currently correct.